qemu/tests/nvdimm_negative.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import virt_vm
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Nvdimm option align negative test:
+    1) Boot vm with incorrect align options
+    2) Check if can get the expected qemu output
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    params['start_vm'] = 'yes'
+    error_msg = params.get('error_msg', '')
+    try:
+        vm.create(params=params)
+        output = vm.process.get_output()
+    except virt_vm.VMCreateError as e:
```

Comment:
Do we need to distinguish the different behaviour for different qemu version ? Create vm failed before QEMu 5.2.

Reply:
I don't think so. As long as no core dump and a corresponding message is prompted, I think it's fine.